provider/in_place_upgrade_base.py

```diff
@@ -109,6 +109,11 @@ def pre_upgrade_whitelist(self, test):
         Fix known issues after executing pre-upgrade
         """
         try:
+            # Leapp and grubby's version
+            s, le = self.session.cmd_status_output("rpm -qa|grep ^leapp")
+            test.log.info("leapp version: %s", str(le))
+            s, gr = self.session.cmd_status_output("rpm -qa|grep grubby")
```

Comment:
```suggestion
            gr = self.session.cmd_output("rpm -q grubby")
```

Reply:
Updated, please help to review it again ! Thanks @PaulYuuu